staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1369,29 +1369,44 @@ type Condition struct {
 	// ---
 	// Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be
 	// useful (see .node.status.conditions), the ability to deconflict is important.
+	// The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
 	// +required
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*/)?(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])$`
```

Comment:
you sure you need the `\\`?  Can you make a proof PR we can play with?

Reply:
fixed